app/presenters/fee/base_fee_presenter.rb

```diff
@@ -57,7 +57,8 @@ def quantity
     # if the error is that the user has typed a decimal when it should be an integer,
     # we want to preserve the decimal value and display on the error page
     #
-    if fee.quantity_is_decimal? || fee.errors[:quantity].include?('integer')
+    if fee.quantity_is_decimal? ||
+       fee.errors[:quantity].include?('You must specify a whole number for this type of fee')
```

Comment:
Nitpick: could we use the translation text here instead of hardcoding? 

eg `fee.errors[:quantity].include?(I18n.t(en.<path to translation>))`

This would mean that if the error wording changes in future we wouldn't need to update this file.

Reply:
Thanks @mpw5 for your suggestion. this is not a single instance and a common pattern throughout the code base. I would like to discuss with the team and get a decision on how should we proceed to get a uniform coding practice. Thanks again and we will address this before or within the [ENDGAME](https://dsdmoj.atlassian.net/browse/CFP-358)